loopchain/channel/channel_service.py

```diff
@@ -509,7 +541,7 @@ def __subscribe_to_peer_list(self):
         peer_request = loopchain_pb2.PeerRequest(
             channel=ChannelProperty().name,
             peer_target=ChannelProperty().peer_target,
-            peer_id=ChannelProperty().peer_id, group_id=ChannelProperty().group_id,
+            peer_id=ChannelProperty().peer_id, group_id=ChannelProperty().peer_id,
```

Comment:
This group_id should be deleted.

Reply:
`group_id` is partially removed here. Group ids related to grpc proto will be updated at the next pull request.